tests/expressions_case/tests.py

```diff
@@ -1148,6 +1148,47 @@ def test_m2m_reuse(self):
             lambda x: x[1:]
         )
 
+    def test_group_by_on_empty_case(self):
+        # Test without default value
```

Comment:
Comments should state the expected behavior and omit prefixes like "Ensure that" / "Test" since all tests ensure things. This guideline is from [Python coding style](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/coding-style/#python-style).

Reply:
Good point!